### DISCUSSION TITLE
Add mod presets to multiplayer

### DISCUSSION THREAD

**Discussion by shertowns** [ppy/osu#28464] [Ideas]
For some reason mod presets are only available in singleplayer?

**Comment by bdach** [comment]
This is only straightforward to implement when selecting required mods during adding/editing of a playlist item. Everywhere else it's more complicated:

- For user mods, only the presets that _can_ be presumably selected should probably be (and how to handle the required/allowed mod split)?
- Free mod selection should not get the same presets because it's a different notion (you can select incompatible free mods for a playlist item, but the user that plays the beatmap in the end has to select a valid compatible combination out of those)/
